### DISCUSSION TITLE
Can't open Github Pages' repositories

### DISCUSSION THREAD

**Discussion by STRlantian** [community/community#32498] [Other Feature Feedback, Questions, & Ideas]
My Github Page repository (https://github.com/Strlantian/strlantian.github.io) is unable to access. I tried to use the public computer in my school then it worked. So why I can't view or control my page repository? I tried to use git but it shows repository not exist. I tried to create a new page repository but it shows repository exists. However I am able to access my Github Page (strlantian.github.io)
I tried to view others' Github Page repositories but I couldn't either. I tried to ask other but some of them could.
![image](https://user-images.githubusercontent.com/68187826/189470671-5b7a418c-71e7-47b7-8365-6f1a5cea1c20.png)

![image](https://user-images.githubusercontent.com/68187826/189470674-cd9c8f00-bae0-4f33-b317-3c751292321f.png)

**Comment by radar-blip** [comment]
Opens fine here. That image you put up, thats all that is on the page, and in your index.html. 
Did you expect something more?

**Reply by STRlantian** [reply]
I wrote a new html file last week and I want to commit but git said could not find repository. Then I tried to open the repository and it showed like this. I wonder if there is anyone has the same situation and what's the solution

**Reply by radar-blip** [reply]
You're using the wrong link, try this: https://strlantian.github.io/   
That is your page right?

**Reply by STRlantian** [reply]
Yes
But the problem is I can't open or control the repository page

**Reply by radar-blip** [reply]
Really? What if you click on your own icon there. It should open your profile page.

**Reply by STRlantian** [reply]
Yes it opened my profile. And my repositories are all able to access except the page repository

**Reply by radar-blip** [reply]
It is here: https://github.com/Strlantian/strlantian.github.io 
I can see it. 😕

**Reply by STRlantian** [reply]
Yes I know. Many people can. But I got a Github 404

**Reply by radar-blip** [reply]
Well thats odd. I dont know what the problem would be.

**Comment by jfullstackdev** [comment]
this is the repo even in Incognito mode

![image](https://user-images.githubusercontent.com/47092464/189491896-39c4df78-f618-493c-9307-c4e832c9fce0.png)

this is the hosted site

![image](https://user-images.githubusercontent.com/47092464/189491903-2ebda906-5071-4292-bb65-2b0a320abf10.png)

if you are accessing that, say, in your school, maybe it's not allowed, if you are using your computer, you might have installed extension, you want to make sure it's not affecting it by logging in Incognito mode or use a fresh regular tab, maybe Edge too

for those who can't load the page, open the Developer Tools and see what kind of error is that, maybe details from there will help

**Reply by STRlantian** [reply]
Well I do use some extensions but they were installed many years ago. And I believe that I can't open my page repository page since at most 1 months ago. So I think it's not a problem of the extensions'
